checks.d/zfs.py

```diff
@@ -0,0 +1,232 @@
+#!/usr/bin/python
+from __future__ import division
+from checks import AgentCheck
+import subprocess
+import re
```

Comment:
Coud you organize your imports the same way it's done in other checks please ?
e.g. https://github.com/DataDog/dd-agent/blob/master/checks.d/redisdb.py#L4-L13